main.js

```diff
@@ -8,9 +8,42 @@ require('./src/js/permalinks');
 require('./src/js/gist-it');
 require('o-header');
 
-(function(){
-	"use strict";
-	document.addEventListener("DOMContentLoaded", function() {
-		document.dispatchEvent(new CustomEvent('o.DOMContentLoaded'));
+// Hack: Array.from is not in the polyfill service's default set.  Once it is this can be removed.
```

Comment:
Add it to the origami.json and it should be fine. I believe we agreed we can use stuff that is polyfilled even if it's not in the default set